packages/flutter/lib/src/gestures/hit_test.dart

```diff
@@ -9,6 +9,12 @@ abstract class HitTestable { // ignore: one_member_abstracts
   void hitTest(HitTestResult result, Point position);
 }
 
+/// An object that can dispatch events.
+abstract class HitTestDispatcher { // ignore: one_member_abstracts
```

Comment:
Yeah, that was my original name here too. I renamed it to this so that it was more consistent with the other related interfaces. (HitTestTarget would be EventTarget by Web standards too.)